pkg/karmadactl/promote/promote.go

```diff
@@ -127,6 +135,9 @@ type CommandPromoteOption struct {
 
 // AddFlags adds flags to the specified FlagSet.
 func (o *CommandPromoteOption) AddFlags(flags *pflag.FlagSet) {
+	flags.BoolVar(&o.AutoCreatePropagationPolicy, "auto-create-propagation-policy", true, "Create a PropagationPolicy automatically")
+	flags.StringVar(&o.PropagationPolicyName, "propagation-policy-name", "",
```

Comment:
Could we only specify `propagation-policy-name`？When the name is nil, auto-gen the name, otherwise using the specific name.

What do you think? @LronDC @jwcesign

Reply:
You mean leave the pp creation automatically, just let the user decide its name, do I get it correctly?

Reply:
Do you mean that there is only one option for `propagation-policy-name`? If that's the case, then the policy will be automatically created, which doesn't solve the issue of manual creation.

Reply:
Got it, it has two phase.

I think it is ok.